### DISCUSSION TITLE
Are preloads supposed to work in alpha11?

### DISCUSSION THREAD

**Discussion by fiammybe** [ImpressCMS/impresscms#1161] [General]
Hi,
I have a impresscms2 alpha 11 setup running, and I added my ga4 analytics preload to it, to see if it also works under 2.0. The preload works as-is under 1.4.2.

The preload is quite stupid :-) It just adds 2 javascript tags to the head section of the page.

Under 2.0, I see no output whatsoever. Are preloads supposed to work in the current version?

**Comment by MekDrop** [comment]
yes, if you register it in composer.json with "preloads" tag

f.e.:
```
  "\\IcmsPreloadAdsense": {
        "tags": [
          "preload"
        ]
      },
```

**Comment by fiammybe** [comment]
oh, so that needs adapting composer.json and running composer from the command line. Not ideal in any way. 
Thanks for the reply!
